src/main/ml-modules/root/com.marklogic.smart-mastering/matcher.xqy

```diff
@@ -98,14 +108,16 @@ declare function matcher:find-document-matches-by-options(
  : @param $start  starting index for potential match results (starts at 1)
  : @param $page-length  maximum number of results to return in this call
  : @param $include-matches  whether the response should list the matched properties for each potential match
+ : @param $filter-query  a cts:query used to filter the matched results
```

Comment:
"a cts:query used to restrict matches to a set, such as a specific entity type or collection"